### PR TITLE
Improving package.json file

### PR DESCRIPTION
**Pull request by vcapretz**
As described in each commit, this PR should improve the package.json file by separating each config in a proper file. 

It also adds some new information as license and repo keys

### CODE REVIEW

**Comment by GitCop** [comment]
Os seguintes commits de seu Pull Request precisam de ajustes no título:

 * Commit: a46f48b27b238374e5762cd3d8bf9e5dfcf01063
  * Bons títulos tem no máximo até 50 caracteres. Para adicionar mais detalhes use o corpo!

O guia completo de estilo de commits está disponível em https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by felquis** [comment]
Running the tests, now it shows this message:

```txt
 PASS  stories/storyshots.test.js
  ● Console

    console.info node_modules/@storybook/react/dist/server/babel_config.js:73
      => Loading custom .babelrc

```

**Comment by vcapretz** [comment]
@felquis it is a known issue: https://github.com/storybooks/storybook/issues/2123, comes from a log in storybook, I don't know if we can do something in our project to prevent it

File: webapp/.babelrc [hunk 1]
```diff
@@ -2,4 +2,4 @@
   "presets": [
     "react-app"
   ]
-}
\ No newline at end of file
+}
```
Comment (felquis) [inline webapp/.babelrc]: ?

File: webapp/package.json [hunk 2]
```diff
@@ -2,6 +2,13 @@
   "name": "greenphoenix",
   "version": "0.1.0",
   "private": true,
+  "license": "(CC-BY-4.0 OR Apache-2.0)",
```
Comment (felquis) [inline webapp/package.json]: This commit should be called "chore: add license and repository url" because "package-json: add new info to package-json file" does't describe the license addition.

File: webapp/.babelrc (hunk 1 above)
Comment (vcapretz) [reply webapp/.babelrc]: newline at the end of the file 😜

File: webapp/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  moduleFileExtensions: [
+    "web.js",
+    "js",
+    "json",
+    "web.jsx",
+    "jsx",
+    "node"
+  ]
+};
```
Comment (felquis) [inline webapp/jest.config.js]: Your commit is `jest: remove jest from package json` does it mean we don't use jest anymore? We can call it `chore: move jest config to jest.config.js`.

File: webapp/package.json [hunk 4]
```diff
@@ -78,43 +78,10 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "lint": "eslint './{src,stories}/**/*.js' && stylelint './{src,stories}/**/*.css'",
-    "test": "node scripts/test.js --env=jsdom",
```
Comment (felquis) [inline webapp/package.json]: This change is not related to your commit message.

File: webapp/.stylelintrc [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      {
+        "ignoreAtRules": [
+          "each",
+          "mixin",
+          "define-mixin"
+        ]
+      }
+    ]
+  }
+}
```
Comment (felquis) [inline webapp/.stylelintrc]: Same as before`stylelint: remove config file from package json`, you have not removed this config, you moved it to the file .stykelintrc. I'd call this message "chore: move stylelint config to .stylelintrc"

File: webapp/package.json (hunk 2 above)
Comment (vcapretz) [reply webapp/package.json]: done

File: webapp/jest.config.js (hunk 3 above)
Comment (vcapretz) [reply webapp/jest.config.js]: done

File: webapp/.babelrc [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "react-app"
+  ]
+}
```
Comment (felquis) [inline webapp/.babelrc]: Here the same tip is valid for the commit message.

File: webapp/.eslintrc.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+      2,
+      {
+        camelCase: true
+      }
+    ]
+  },
+  plugins: [
+    "css-modules"
+  ],
+};
```
Comment (felquis) [inline webapp/.eslintrc.js]: The same here for the commit message.

File: webapp/.stylelintrc (hunk 5 above)
Comment (vcapretz) [reply webapp/.stylelintrc]: done

**Comment by felquis** [comment]
We call `chore` all the changes that doesn't change or add the behavior that the final user will notice.

**Comment by felquis** [comment]
If your change changes something, and the final user will not notice your change, only developers for example, that change is a `chore`. This is what we follows here at least :)

File: webapp/package.json (hunk 2 above)
Comment (derekstavis) [reply webapp/package.json]: Não é OR...

**Comment by derekstavis** [comment]
Eu havia dito que não gostava de dotfiles e que idealmente esses arquivos deveriam estar dentro do diretório config... Anyway, a gente refatora depois 👍

**Comment by felquis** [comment]
Boa sugestão Derek.
